src/measure_theory/integral/set_integral.lean

```diff
@@ -160,29 +150,45 @@ begin
   ... = ∫ x in s, f x ∂μ : by simp
 end
 
+lemma tendsto_set_integral_of_monotone {ι : Type*} [encodable ι] [semilattice_sup ι]
+  {s : ι → set α} {f : α → E} (hsm : ∀ i, measurable_set (s i))
+  (h_mono : monotone s) (hfi : integrable_on f (⋃ n, s n) μ) :
+  tendsto (λ i, ∫ a in s i, f a ∂μ) at_top (𝓝 (∫ a in (⋃ n, s n), f a ∂μ)) :=
+begin
+  have hfi' : _ < _ := hfi.2,
```

Comment:
IMHO, you could maybe sometimes try to optimize your code first for the human reader, and only secondly for Lean, instead of the opposite order (a.k.a., ultimate golfing is not always a good idea :-) Could you spell out the underscores here?